examples/Dockerfile.web

```diff
@@ -12,7 +12,7 @@ ENV PORT ${PORT}
 #   listed in package-lock.json, and does not update either the package-lock.json or the package.json file.
 # Our production dependancies are now installed.
 COPY --chown=node:node package*.json ./
-RUN npm ci
+RUN npm ci --production

```

Comment:
I can't find any mention of `npm ci` accepting a `production` flag in the documentation. I always assumed it would only install production dependencies anyway, but I can't find any hints as to whether that's true.